src/store/index.js

```diff
@@ -13,7 +13,8 @@ export default new Vuex.Store({
     collections: [],
     records: [],
     collectionName: '',
-    totalNumberOfResults: null
+    totalNumberOfResults: null,
+    recordList: []
```

Comment:
what is recordList?

Reply:
I was trying a clean way to represent nested documents recursively. Again, forgot to clean up afterwards.